### PR TITLE
[PREVIEW]OGM-1047 exclude database recreation

### PR DESCRIPTION
**Pull request by schernolyas**
Neo4j database is recreated for each test. It is too expensive operation. The commit clear currect database only.

### CODE REVIEW

**Comment by DavideD** [comment]
Is the builder faster with this approach?

**Comment by schernolyas** [comment]
Hi @DavideD !

Hm ... In theory, yes. Because we do not database restart.

**Comment by DavideD** [comment]
> Hm ... In theory, yes. Because we do not database restart.
???
Did you try it?

File: neo4j/src/test/java/org/hibernate/ogm/datastore/neo4j/utils/EmbeddedNeo4jTestHelperDelegate.java [hunk 1]
```diff
@@ -70,18 +68,8 @@ public GridDialect getDialect(DatastoreProvider datastoreProvider) {
 
 	@Override
 	public void dropDatabase(DatastoreProvider datastoreProvider) {
-		( (Stoppable) datastoreProvider ).stop();
```
Comment (DavideD) [inline neo4j/src/test/java/org/hibernate/ogm/datastore/neo4j/utils/EmbeddedNeo4jTestHelperDelegate.java]: How come you removed the calls to datastoreProvider.stop();?

**Comment by DavideD** [comment]
It looks alright but I have 3 failures:

[ERROR] Errors:                                                                                                                     
[ERROR]   CompensationSpiJpaTest.onRollbackTriggeredThroughJtaPresentsAppliedInsertOperations:64 ? Persistence             
[ERROR]   JPAAPIWrappingTest.testWrappedStandalone:38 ? Persistence Unable to build enti...
[ERROR]   JPAJTATest.testBootstrapAndCRUD:39 ? Persistence Unable to build entity manage...

http://ci.hibernate.org/job/hibernate-ogm-PR/1468/testReport/

**Comment by DavideD** [comment]
I'm closing this PR for now, feel free to send a new one when you solved the issue about the 3 failures

**Pull request closed without merging**